cmd/openshift-tests/e2e.go

```diff
@@ -43,6 +43,15 @@ var staticSuites = []*ginkgo.TestSuite{
 			return strings.Contains(name, "[Suite:openshift/conformance/serial")
 		},
 	},
+	{
+		Name: "openshift/multitenant/conformance",
+		Description: templates.LongDesc(`
+		Only the portion of the openshift/multitenant test suite.
+		`),
+		Matches: func(name string) bool {
+			return !strings.Contains(name, "[Feature:NetworkPolicy]")
```

Comment:
You need to include the rule from `openshift/conformance` as well. So

    return strings.Contains(name, "[Suite:openshift/conformance/") && !strings.Contains(name, "[Feature:NetworkPolicy]")